app/uk/gov/hmrc/gform/formstatistics/FormStatisticsModule.scala

```diff
@@ -40,10 +38,7 @@ class FormStatisticsModule(
     new Repo[Form](
       "forms",
       mongoModule.mongoComponent,
-      _._id.value,
```

Comment:
I think you will have the same error. You create a new PlayMongoReposiory with empty indexes and it runs default ensureIndexes on construction. `ensureIndexes` then checks that existing indexes in `gform.forms` are not equal to empty indexes and throws the same error.

Reply:
Not sure what to do here. May be we need to just duplicate the same indexes in the Statistics repo. Or somehow have it set up in one place.